cli/dcoscli/data/help/package.txt

```diff
@@ -68,6 +71,9 @@ Options:
         Print a short description of this subcommand.
     --options=<file>
         Path to a JSON file that contains customized package installation options.
+    --output-directory=<output-directory>
+        Path to the directory where the data should be stored.
+        Defaults to target/ underneath the directory of the DC/OS Package Build Definition.
```

Comment:
@tamarrow I would too, and I did. But I got some feedback at the standup that it would be better in target/ so I changed it.

@jsancio How about 'the directory of <build-definition>'?